src/herbpy/action/pouring.py

```diff
@@ -0,0 +1,72 @@
+import openravepy, time, prpy, math, numpy
+from prpy.action import ActionMethod
+
+@ActionMethod
+def MoveCupAndPour(robot, table, manip_pitcher, cup, pitcher):
+    """
+    @param robot The robot performing the pouring
+    @param table The table the objects are on
+    @param manip_pitcher The arm that is manipulating the pitcher
+    @param cup The cup to pour into
+    @param pitcher The pitcher to pour
+    """	
+
+    manip_cup = robot.right_arm
```

Comment:
Please take the `manipulator` as an argument. I would also prefer if it defaulted to `GetActiveManipulator()`, instead of the `right_arm`.